doc/api/sqlite.md

```diff
@@ -429,13 +429,23 @@ exception.
 | `TEXT`    | {string}             |
 | `BLOB`    | {Uint8Array}         |
 
-## SQLite constants
+## `sqlite.constants`
 
-The following constants are exported by the `node:sqlite` module.
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object}
+
+An object containing commonly used constants for SQLite operations.
+
+### SQLite constants
+
+The following constants are exported by the `sqlite.constants` object.
 
-### SQLite Session constants
+#### SQLite Session constants
 
-#### Conflict-resolution constants
+##### Conflict-resolution constants
```

Comment:
YMMV, but might it be clearer to remove a level of nesting here?

```diff
- #### SQLite Session constants
- 
- #### Conflict-resolution constants
+ ### Conflict-resolution constants
```

I don't get the sense that this many layers of headers improves readability, and equally I don't think that there are ever going to be so many properties here to warrant sub-sub-classifying them.

Reply:
It looks good. I don't know what the maintainers think.

Reply:
I don't really have an opinion on this.

Reply:
I think, removing extra level would be better.
We can always add extra nesting once there are more constants added.

Reply:
Sounds good. Thanks guys.